envi/archs/i386/opcode86.py

```diff
@@ -354,7 +354,7 @@
 (0, 0, ARG_NONE, ARG_NONE, ARG_NONE, 0, 0, 0, 0, 0),  
 (0, 0, ARG_NONE, ARG_NONE, ARG_NONE, 0, 0, 0, 0, 0),  
 (0, 0, ARG_NONE, ARG_NONE, ARG_NONE, 0, 0, 0, 0, 0),  
-(0, 0, ARG_NONE, ARG_NONE, ARG_NONE, 0, 0, 0, 0, 0),  
+(0, INS_OTHER, ADDRMETH_I | OPTYPE_b | OP_R, ADDRMETH_I | OPTYPE_b | OP_R, ARG_NONE, cpu_80386, "vpcext", 0, 0, 0),  
```

Comment:
could you please create `cpu_VIRTUALPC = 0x0000f000` in `envi/archs/i386/opconst.py` and use that here instead of `cpu_80386`?  
this will help us delineate that this instruction doesn't actually exist in any defined x86 architecture.  this instruction is only supported in Virtual PC, correct?

Reply:
Thank you for this feedback :)